From 73bb7009f980da3e898b590be06328703235c8ee Mon Sep 17 00:00:00 2001
From: Arthur Su <arthur.su@tp-link.com>
Date: Fri, 16 Jul 2021 09:43:25 +0000
Subject: [PATCH] ipq40xx: change EC420-G1 patch_mac method

deprecate get_tip command, because manufacturing data aren't binrary
raw.

Signed-off-by: Arthur Su <arthur.su@tp-link.com>
---
 .../base-files/etc/hotplug.d/firmware/11-ath10k-caldata       | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata b/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
index 428002acf3..38f248fb6e 100644
--- a/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
+++ b/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
@@ -129,7 +129,7 @@ case "$FIRMWARE" in
 	tp-link,ap2220 |\
 	tp-link,ec420-g1)
 		ath10kcal_extract "0:ART" 36864 12064
-		ath10kcal_patch_mac_crc $(macaddr_add "$(get_tip mac_address)" 1) # 5G, wlan0
+		ath10kcal_patch_mac_crc $(macaddr_add "$(cat /sys/class/net/eth0/address)" 1) # 5G, wlan0
 		;;
 	esac
 	;;
@@ -200,7 +200,7 @@ case "$FIRMWARE" in
 	tp-link,ap2220 |\
 	tp-link,ec420-g1)
 		ath10kcal_extract "0:ART" 4096 12064
-		ath10kcal_patch_mac_crc $(macaddr_add "$(get_tip mac_address)" 9) #2.4G, wlan1
+		ath10kcal_patch_mac_crc $(macaddr_add "$(cat /sys/class/net/eth0/address)" 9) #2.4G, wlan1
 		;;
 	esac
 	;;
-- 
2.20.1

